examples/auth_example/auth_example_server/migrations/serverpod_auth/migration_registry.json

```diff
@@ -1 +1,8 @@
-{"className":"DatabaseMigrationRegistry","data":{"migrations":["20231114162745"]}}
\ No newline at end of file
+{
+  "className": "DatabaseMigrationRegistry",
```

Comment:
Can we store this without the class name as we know that it is always of the "DatabaseMigrationRegistry" class. It would be more readable with only migrations part.

```json
"migrations": [
  "20231115140540"
]
```

JSON may also not be an optimal format here as we would get a line change on the last entry of the list if we add a comma (which can cause unwanted confilcts). I would prefer a text file or YAML.

Reply:
Remove the class context information for both the migration files and for the migration registry.

Keeping it as json for now since we might want to encode more information in this file. But I agree that Yaml might be a better format if the requirements of the file stay the same :)

Will look this over once the migrations system is a bit more polished.